README.md

```diff
@@ -1,2 +1,23 @@
-# duckduckhack.com
+# Welcome to DuckDuckHack!
+
 This repo contains the static content used to build [DuckDuckHack.com](DuckDuckHack.com)
+
+## Setting up
+
+This project requires that you have [npm](https://www.npmjs.com/), [bower](https://bower.io/), [grunt](https://gruntjs.com/) and [jekyll](https://jekyllrb.com/) installed globally. You can set up and build the website using the following steps:
+
+```
+$ git clone https://github.com/duckduckgo/duckduckhack.com
+$ cd duckduckhack.com/
+$ npm i
+$ bower install
+$ grunt
+$ jekyll s
```

Comment:
Jekyll is technically optional. Grunt builds the compiled JS, and the HTML is static so you can just open issues.html in your browser after running grunt and it works fine 👍